fdbserver/QuietDatabase.actor.cpp

```diff
@@ -966,8 +966,11 @@ ACTOR Future<Void> waitForQuietDatabase(Database cx,
                                         int64_t maxStorageServerQueueGate = 5e6,
                                         int64_t maxDataDistributionQueueSize = 0,
                                         int64_t maxPoppedVersionLag = 30e6,
-                                        int64_t maxVersionOffset = 1e6) {
-	state QuietDatabaseChecker checker(isGeneralBuggifyEnabled() ? 4000.0 : 1500.0);
+                                        int64_t maxVersionOffset = 1e6,
+                                        double quiescentWaitTimeout = 0) {
```

Comment:
Why do we want to add this quiescentWaitTimeout? Thanks!

Reply:
I see. According to the definition of QuietDatabaseChecker indicating the input is `maxDDRunTime`. If DD timed out, there is a `ddGotStuck` assertion failure. So, probably better to change `quiescentWaitTimeout` to `maxDDRunTime`.

The `maxDDRunTime` is different from the option of quiescentWaitTimeout for consistency check workload.